Tasks/VsTest/taskinputparser.ts

```diff
@@ -33,6 +34,11 @@ export function getDistributedTestConfigurations() {
     }
     tl._writeLine(tl.loc('dtaNumberOfAgents', dtaConfiguration.numberOfAgentsInPhase));
 
+    const useVsTestConsole = tl.getVariable('UseVsTestConsole');
+    if (useVsTestConsole) {
+        dtaConfiguration.useVsTestConsole = useVsTestConsole;
+    }  
```

Comment:
Just confirming: that customers now will not be able to use vstestconsole for test execution until and unless they set task variable UseVsTestConsole to true in their release definiton.